ZUGFeRD/InvoiceDescriptor22UBLWriter.cs

```diff
@@ -51,7 +51,7 @@ public override void Save(InvoiceDescriptor descriptor, Stream stream, ZUGFeRDFo
                 { "ext", "urn:oasis:names:specification:ubl:schema:xsd:CommonExtensionComponents-2" },
                 { "xs", "http://www.w3.org/2001/XMLSchema" }
             };
-            if (this.Descriptor.Type == InvoiceType.Invoice)
+            if (this.Descriptor.Type == InvoiceType.Invoice || this.Descriptor.Type == InvoiceType.Correction)
```

Comment:
can you please add brackets around the parts of the if clause (consistent code style)?